drivers/interrupt_controller/intc_exti_stm32.c

```diff
@@ -113,7 +113,7 @@ void stm32_exti_enable(int line)
 {
 	int irqnum = 0;
 
-	if (line >= ARRAY_SIZE(exti_irq_table)) {
+	if (line >= (int)ARRAY_SIZE(exti_irq_table)) {
```

Comment:
`line` should never be a negative number, so its type should be `unsigned int`.

Reply:
That was my initial inclination, but I was afraid to adjust an API without consulting with a `Member` first.

It appears the only reference to this API is in `drivers/gpio/gpio_stm32.c`, and it passes in a `uint8_t` as a parameter. It should also maintain ABI compatibility, because `unsigned int` and `int` are the same size. Therefore, this seems like a low risk modification to the API.